### ISSUE TITLE
Instant memory leak when pasting cut files in Nautilus

### ISSUE THREAD

**Issue by mdziekon** [GSConnect/gnome-shell-extension-gsconnect#676]
**Describe the bug**

Today I've noticed strange behaviour of my OS, where it would out of the blue get really slow (all interactions got slow). It turned out that the system's memory was full, so swapping has started making the entire OS unusable. The offending process turned out to be ``gjs`` running an instance of GSConnect's ``services/daemon.js``. After a couple of seconds, it seems that GNOME would automatically kill that process, so the high memory usage would drop down to reasonable levels. Then, after a while, Ubuntu would report that the extension crashed, however it would also say that it was unable to have the crash report due to low memory available.

Later I've discovered that this behaviour was triggered by cutting & pasting files in Nautilus. It doesn't matter what the file contains or how big it is, pasting the file (just cutting it didn't do anything until you actually pasted it) would clog the memory in a matter of seconds.

I've investigated further, disabling all other extensions, disabling all GSConnect plugins, even disconnecting all devices. None of this matters, the problem still persists. The only thing that fixes the problem is to disable the GSConnect extension entirely.

**Steps To Reproduce:**

1. Open Nautilus at any location where you have a file that can be moved
2. Cut the file (CTRL + X)
3. Paste the file, can be at the same location or a different one (CTRL + V)
4. System's memory usage instantly goes up to 100%, clogging it and causing the extension to eventually crash and restart.

**Expected behavior**

The memory usage should **not** go up to 100%.

**Support Log**

```
GSConnect Version: 27
GSConnect Install: user
GJS: 15205
XDG_SESSION_TYPE: x11
GDMSESSION: ubuntu
--------------------------------------------------------------------------------
-- Logs begin at Fri 2019-05-03 15:51:23 CEST, end at Sat 2019-10-19 01:01:04 CEST. --
paź 19 01:00:01 gnome-shell[25414]: Object Meta.Background (0x558032178e60), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #0 0x55802bb992e0 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #1 0x55802bb99258 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #2 0x55802bb991b8 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #3 0x55802bb99098 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:00:01 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:01 gnome-shell[25414]: Object Meta.Background (0x558032178e60), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #0 0x55802bb991c0 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #1 0x55802bb99138 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #2 0x55802bb99098 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #3 0x7ffc918c83f0 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:00:01 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:01 gnome-shell[25414]: Object Meta.Background (0x558032178e60), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #0 0x55802bb99120 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #1 0x55802bb99098 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #2 0x7ffc918c7b30 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #3 0x7ffc918c83f0 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:00:01 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:00:01 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:26 gjs[28919]: GtkDialog mapped without a transient parent. This is discouraged.
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:42 gjs[30048]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:00:47 gjs[28919]: GtkDialog mapped without a transient parent. This is discouraged.
paź 19 01:00:51 nautilus[19619]: gtk_revealer_set_reveal_child: assertion 'GTK_IS_REVEALER (revealer)' failed
paź 19 01:01:02 kernel: gjs[30048]: segfault at 0 ip 00007fa1fd8cf258 sp 00007ffe486fc0f0 error 4 in libgjs.so.0.0.0[7fa1fd8a7000+c7000]
paź 19 01:01:02 kernel: Code: 0f 84 b7 fc ff ff 8d 45 ff 48 8d 6c 24 28 45 31 ff 48 8d 04 c5 08 00 00 00 48 89 44 24 08 0f 1f 40 00 48 8b b4 24 e0 00 00 00 <4a> 8b 04 3b 41 b8 01 00 00 00 48 89 e9 4c 89 f2 4c 89 e7 4c 01 fe
paź 19 01:01:03 dbus-daemon[2806]: [session uid=1000 pid=2806] Monitoring connection :1.1576 closed.
paź 19 01:01:03 gjs[28919]: ../../../../gobject/gsignal.c:2641: instance '0x561ff14a5590' has no handler with id '86059'
paź 19 01:01:03 gjs[28919]: ../../../../gobject/gsignal.c:2641: instance '0x561ff14a5590' has no handler with id '86060'
paź 19 01:01:03 gjs[28919]: ../../../../gobject/gsignal.c:2641: instance '0x561ff14a5590' has no handler with id '86061'
paź 19 01:01:03 gjs[28919]: ../../../../gobject/gsignal.c:2641: instance '0x561ff14a5590' has no handler with id '86062'
paź 19 01:01:03 dbus-daemon[2806]: [session uid=1000 pid=2806] Activating service name='org.gnome.Shell.Extensions.GSConnect' requested by ':1.1485' (uid=1000 pid=25414 comm="/usr/bin/gnome-shell " label="unconfined")
paź 19 01:01:03 gnome-shell[25414]: Object Meta.Background (0x5580326839e0), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:01:03 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #0 0x55802bb992e0 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #1 0x55802bb99258 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #2 0x55802bb991b8 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #3 0x55802bb99098 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #0 0x55802bb991c0 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #1 0x55802bb99138 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #2 0x55802bb99098 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #3 0x7ffc918c83f0 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:01:03 gnome-shell[25414]: Object Meta.Background (0x5580326839e0), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: == Stack trace for context 0x55802b6f34d0 ==
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #0 0x55802bb99120 i   resource:///org/gnome/shell/ui/background.js:718 (0x7f59b9d0f4d8 @ 22)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #1 0x55802bb99098 i   resource:///org/gnome/shell/ui/tweener.js:112 (0x7f59d81c8e68 @ 37)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #2 0x7ffc918c7b30 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:208 (0x7f59d81d2b38 @ 54)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #3 0x7ffc918c83f0 b   resource:///org/gnome/gjs/modules/tweener/tweener.js:337 (0x7f59d81d2bc0 @ 1626)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #4 0x55802bb99008 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:350 (0x7f59d81d2c48 @ 100)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #5 0x55802bb98f90 i   resource:///org/gnome/gjs/modules/tweener/tweener.js:365 (0x7f59d81d2cd0 @ 10)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #6 0x7ffc918c9060 I   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f59d81d2230 @ 386)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #7 0x55802bb98f00 i   resource:///org/gnome/shell/ui/tweener.js:244 (0x7f59d81cf808 @ 159)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #8 0x7ffc918c9c40 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f59d81b5de0 @ 71)
paź 19 01:01:03 org.gnome.Shell.desktop[25414]: #9 0x55802bb98e80 i   resource:///org/gnome/shell/ui/tweener.js:219 (0x7f59d81cf780 @ 15)
paź 19 01:01:03 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:03 gnome-shell[25414]: Object Meta.Background (0x5580326839e0), has been already deallocated - impossible to access it. This might be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
paź 19 01:01:03 gnome-shell[25414]: g_object_run_dispose: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:03 dbus-daemon[2806]: [session uid=1000 pid=2806] Successfully activated service 'org.gnome.Shell.Extensions.GSConnect'
paź 19 01:01:03 org.gnome.Shell.Extensions.GSConnect[30325]: [/service/components/input.js:_prepareController:296]: "Falling back to Atspi"
paź 19 01:01:04 gjs[30325]: g_variant_new_string: assertion 'string != NULL' failed
paź 19 01:01:04 dbus-daemon[2806]: [session uid=1000 pid=2806] Connection :1.1582 (uid=1000 pid=30325 comm="gjs /home/mdziekon/.local/share/gnome-shell/extens" label="unconfined") became a monitor.
paź 19 01:01:04 org.gnome.Shell.Extensions.GSConnect[30325]: [/service/components/mpris.js:_addPlayer:168]: "Adding MPRIS Player Rhythmbox"
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 dbus-daemon[1480]: [system] Activating via systemd: service name='org.freedesktop.hostname1' unit='dbus-org.freedesktop.hostname1.service' requested by ':1.279' (uid=1000 pid=28919 comm="gjs /home/mdziekon/.local/share/gnome-shell/extens" label="unconfined")
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 gjs[30325]: g_object_unref: assertion 'G_IS_OBJECT (object)' failed
paź 19 01:01:04 systemd[1]: Starting Hostname Service...
paź 19 01:01:04 dbus-daemon[1480]: [system] Successfully activated service 'org.freedesktop.hostname1'
paź 19 01:01:04 systemd[1]: Started Hostname Service.
```

**System Details:**

 - **GSConnect version:** 27
   - **Installed from:** GNOME Extensions Website
 - **GNOME/Shell version:** 3.28.2
 - **Distro/Release:** Ubuntu 18.04.3 (kernel 5.0.0-31-generic) 

**GSConnect environment (if applicable):**

 - **Paired Device(s):** One Plus One, Dell XPS 9570 (Disconnected)
 - **KDE Connect app version:** 1.13.2
 - **Plugin(s):** doesn't matter, the problem persists even when **ALL** plugins are disabled

**Additional Notes:**

RAM amount: 32 G
SWAP: 16 G

**Comment by andyholmes** [comment]
I have a vague idea what might be causing this, but I'm not very optimistic that it's fixable in 3.28 since this likely is related to Nautilus changing it's clipboard format or worse old XWayland bugs.


*[copied from #653]*

There seem to be quite a few problems happening with GNOME 3.28, and sadly it may just be time to drop support for older versions. I don't have the hard drive space to keep VMs around for all these versions or the time to properly support them, and we've been holding back on newer language features and libraries going back over two years now.

So basically, unless someone volunteers to maintain support for older versions of GNOME, I think we'll drop support for them in v28.

**Comment by andyholmes** [comment]
Closing since this only seems to be reported in Ubuntu 18.04 which we no longer support.

**Comment by abartov** [comment]
This still happens in gnome-shell 3.36.4.  Cutting and pasting hundreds of files causes memory leaks. :(

**Comment by andyholmes** [comment]
Does it happen if you disable the clipboard plugin on all devices?

**Comment by abartov** [comment]
Thanks for the quick response!  The only Extensions I have enabled are "Desktop Icons", "Ubuntu AppIndicators", and "Ubuntu Dock".

**Comment by behrangsa** [comment]
Could this be related? Unfortunately by the time I managed to grab a screenshot a couple of other high memory usage processes died:
![gjs](https://user-images.githubusercontent.com/18451/101944481-e60b3600-3c40-11eb-94ce-3be77786828f.png)

**Comment by andyholmes** [comment]
> Happy to try anything you ask to help triage this.

Try [disabling the Clipboard Plugin](https://github.com/andyholmes/gnome-shell-extension-gsconnect/wiki/Features#Plugins) for all devices, in GSConnect.

**Comment by abartov** [comment]
I'm not sure what GSConnect is, and as I wrote above, am not seeing such a plugin in the gnome-shell settings.  Perhaps I'm commenting on the wrong issue? If so, I apologize.  

One piece of information I can add: whereas the nautilus from my regularly-updated Ubuntu 20.04 (nautilus v 3.36.3) is consistently leaking memory when copying and pasting large amounts of (not-large) files, the **nightly version** (currently 40.alpha-5b0301b0d) I have installed via flatpak **does not**.  I am happy to help debug this further if it helps, but for me personally, switching to nightly worked, so presumably whatever it was has already been corrected on the master branch.
